package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jwplayer",
-  "version": "8.3.0-rc.1",
```

Comment:
This change shouldn't be showing up. You should rebase or make new branch with just the changes for this feature.